### ISSUE TITLE
Don't catch EOFError

### ISSUE THREAD

**Issue by kissgyorgy** [IoT-Inspector/unblob#13]
Click is catching `EOFError` and quitting with an `Aborted!` message.

We are quite often getting this Exception during development, because of the nature of the project, but
we need the traceback very much to make our life easier.
https://click.palletsprojects.com/en/7.x/exceptions/?highlight=standalone_mode

**Comment by QKaiser** [comment]
Current working solution, not sure if this is Pythonic or even right 😸 

```python
@click.option("-v", "--verbose", is_flag=True, help="Verbose mode, enable debug logs.")
@click.pass_context
def build(ctx: click.Context, **kwargs):
    return ctx

def run(files: Tuple[Path], extract_root: Path, depth: int, verbose: bool):
    configure_logger(verbose, extract_root)
    logger.info("Start processing files", count=len(files))
    for path in files:
        process_file(path.parent, path, extract_root, depth)

def main():
    ctx = build(standalone_mode=False)
    run(**ctx.params)


if __name__ == "__main__":
    main()
```

**Comment by kukovecz** [comment]
Thank you @QKaiser! Your solution is working, and gave a basic idea, but we went for (I hope :sweat_smile: ) for a more Pythonic way. 

Basically your solution is creating an empty `click.Command` which does nothing (so it is skipping every `click` exception handling mechanism), then executes its method.

We went with the direct `invoke` calling as the official `click` document [suggests](https://click.palletsprojects.com/en/8.0.x/exceptions/).
